src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2020,4 +2020,8 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
     <value>Attribution</value>
     <comment>giving credit to the projects this utility was based on</comment>
   </data>
+  <data name="FileExplorerPreview_ToggleSwitch_Wrap_Text.Header" xml:space="preserve">
```

Comment:
This should be .Content, not .Header.. that might be the reason why it crashes?

Reply:
I see this has been changed, will mark this review as resolved. Thank you.